lib/fathead/pip/parse.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+
+from bs4 import BeautifulSoup
+
+baseUrl = 'https://pip.pypa.io/en/stable/reference/'
+
+output = "output.txt"
+
+outyo = open(output, "w");
+
+import os
+from os import path
+pages = [f for f in os.listdir('./download/')]
+
+out = {}
+
+for page in pages:
+
+    finaltext = ''
+    hashtable = {}
+    with open('./download/' + page, 'r') as content_file:
+        page_source = content_file.read()
+        soup = BeautifulSoup(page_source,"lxml")
+        namestr = ''
+        name = page[:-5].split('_')
+        for n in name:
+            namestr += n + "-"
+
+        namestr = namestr[:-1]
+
+        pip_show = soup.find("div",{"id":namestr})
+
+        description = pip_show.findAll("div",{"class":"section","id":"description"})
+        usages = pip_show.findAll("div",{"class":"section","id":"usage"})
+
+        for usage in usages:
+            for child in usage:
+                try:
+                    text = child.text.strip()
+                    if(text not in hashtable):
+                        hashtable[text] = 1
+                    else:
+                        continue
+                    if(child.find('div','highlight')):
+                        fin = ''
+
+                        for t in text.split('...'):
+                            fin += t.strip() + '\n'
+
+                        fini = ''
+
+                        for t in fin.split('\n'):
+                            fini += t.strip() + '\n'
+
+                        text = '<pre><code>' + fini +'</pre></code>'
```

Comment:
@karanchahal the closing brackets are wrong -- should be `</code></pre>` -- this is likely causing none of the entries to be installed in our DB 👍